strawberryfields/api/result.py

```diff
@@ -118,7 +116,18 @@ def state(self):
 
     def __repr__(self):
         """String representation."""
-        shots, modes = self.samples.shape
-        return "<Result: num_modes={}, shots={}, contains state={}>".format(
-            modes, shots, self._is_stateful
-        )
+        if self.samples.ndim == 2:
+            # if samples has dim 2, assume they're GBS
```

Comment:
```suggestion
            # if samples has dim 2, assume they're from a standard Program
```
Might not be too accurate to say 'GBS' here, since this applies to _any_ executed `sf.Program` object.